packages/rum-core/src/domain/configuration/configuration.ts

```diff
@@ -115,6 +115,7 @@ export interface RumInitConfiguration extends InitConfiguration {
   trackViewsManually?: boolean | undefined
   /**
    * Enables collection of resource events.
+   * @default true
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why only document this one with a `@default` ?

Reply:
no reasons, fixed!